main/ui/src/main/resources/css/dark_theme.css

```diff
@@ -5,19 +5,19 @@
  ******************************************************************************/
 
 @font-face {
-	src: url('opensans-regular.ttf');
+    src: url('opensans-regular.ttf');
```

Comment:
I think we would end up with a smaller diff, if we use tabs instead of spaces for CSS, too.

And _if_ we decide for spaces we should do some research on what is considered best practice in 2020. For JS two-space indents became the de-facto standard. Not sure about CSS, though.